Lib/opcode.py

```diff
@@ -84,6 +84,7 @@ def pseudo_op(name, op, real_ops):
 def_op('UNARY_NOT', 12)
 
 def_op('UNARY_INVERT', 15)
+def_op('EXIT_INIT_CHECK', 16)
```

Comment:
This needs to be added to the opcodes list in the docs.

Reply:
I'd rather not. It shouldn't appear in any disassembly or traceback.
Once we have some sort of assembler, I want to replace
```
    EXIT_INIT_CHECK
    RETURN_VALUE
```
with
```
    POP_JUMP_IF_NOT_NONE error
    RETURN_VALUE
error:
    PUSH_NULL
    LOAD_CONST TypeError
    LOAD_CONST "__init__() should not return a value"
    CALL 1
    RAISE_VARARGS 1
```
So I expect it to removed before 3.12beta.